cypress/integration/tests/applicationinventory/application/sort.test.ts

```diff
@@ -0,0 +1,93 @@
+/// <reference types="cypress" />
+
+import {
+    login,
+    clickByText,
+    sortAsc,
+    sortDesc,
+    verifySortAsc,
+    verifySortDesc,
+    getTableColumnData,
+} from "../../../../utils/utils";
+import { navMenu, navTab } from "../../../views/menu.view";
+import { controls, applicationinventory, name, tags } from "../../../types/constants";
+
+import * as data from "../../../../utils/data_utils";
+import { ApplicationInventory } from "../../../models/applicationinventory/applicationinventory";
+
+var applicationsList: Array<ApplicationInventory> = [];
+
+describe("Business services sort validations", function () {
+    before("Login and Create Test Data", function () {
+        // Perform login
+        login();
+        var tagsList = ["C++", "COBOL", "Java"];
+        // Create multiple applications with tags
+        for (let i = 0; i < 3; i++) {
+            // Create new application
+            const application = new ApplicationInventory(data.getFullName(), "", "", "", tagsList);
+            application.create();
+            applicationsList.push(application);
+            tagsList.pop();
+        }
+    });
+
+    beforeEach("Persist session", function () {
+        // Save the session and token cookie for maintaining one login session
+        Cypress.Cookies.preserveOnce("AUTH_SESSION_ID", "KEYCLOAK_SESSION");
+
+        // Interceptors
+        cy.intercept("GET", "/api/application-inventory/application*").as("getApplications");
+    });
+
+    after("Perform test data clean up", function () {
+        // Delete the applications created before the tests
+
+        applicationsList.forEach(function (application) {
+            application.delete();
+        });
+    });
+
+    it("Name sort validations", function () {
+        // Navigate to application inventory page
+        clickByText(navMenu, applicationinventory);
+        cy.wait("@getApplications");
+
+        // Sort the application inventory by name in ascending order
+        sortAsc(name);
+        cy.wait(2000);
+
+        // Verify that the application inventory table rows are displayed in ascending order
+        const afterAscSortList = getTableColumnData(name);
+        verifySortAsc(afterAscSortList);
```

Comment:
This function has been updated, it accepts two params now. You need to get unsorted list in all tests now and then pass it as 2nd param in both the asc and desc sort verify functions.